static/app/views/performance/transactionSummary/transactionEvents/eventsTable.tsx

```diff
@@ -103,10 +106,7 @@ class EventsTable extends React.Component<Props, State> {
         action,
       });
 
-      const searchConditions = new MutableSearch(eventView.query);
-
-      // remove any event.type queries since it is implied to apply to only transactions
-      searchConditions.removeFilter('event.type');
+      const searchConditions = normalizeSearchConditions(eventView.query);
```

Comment:
should this be `normalizeSearchConditionsWithTransactionName`? otherwise, transaction filter will also end up being removed here